### PR TITLE
Theme system, with default dark and light themes

### PR DESCRIPTION
**Pull request by iamreyne**
This fork includes a theme system, converting the current dark theme into one css file, and adding a brand new light theme.

Adds one new option to the config file, $themeChoice

### CODE REVIEW

**Comment by ghost** [comment]
Looks good, I'll have to try it out!

File: modules/defaults.php [hunk 1]
```diff
@@ -6,14 +6,17 @@
...
+
 // autorefresh interval for the status webpage in seconds
 $autoRefreshInSeconds = 5;
 
 // Name of your node (default: your hostname)
 $nanoNodeName = gethostname();
 
 // a welcome message shown on top
-$welcomeMsg = '';
+$welcomeMsg = 'Welcome to Nano Node Monitor';
```
Comment (ghost) [inline modules/defaults.php]: I'd prefer no welcome message as default.

File: modules/defaults.php [hunk 2]
```diff
@@ -6,14 +6,17 @@
 
 // ----------- General Variables -----------
 
+// Theme of your Node Monitor (dark or light)
+$themeChoice = 'light';
```
Comment (ghost) [inline modules/defaults.php]: Not sure whether 'light' should be the default. I'd like to keep the current theme as default.

File: templates/index.hbs [hunk 3]
```diff
@@ -1,125 +1,131 @@
...
-        <span class="mr-3">Address</span>
-        <a href="{{nanoNodeAccountUrl}}" target="_blank" class="truncate">{{nanoNodeAccount}}</a>
-        <button id="copyAccount" class="btn btn-sm btn-link" data-clipboard-text="{{nanoNodeAccount}}" title="Copy">
-          <i class="fas fa-clipboard fa-lg"></i>
-        </button>
-      </li>
-    </ul>
-  </div>
+    <div class="col">
+        <h2 class="n-h2-theme">Node Account</h2>
```
Comment (ghost) [inline templates/index.hbs]: Having Node Account as h2 and the other heading as h3 looks weird to me: https://imgur.com/a/1sbYBUs

File: static/css/dark.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
+/**
```
Comment (ghost) [inline static/css/dark.css]: Can you make the links in the 'dark' theme white instead of orange ?

**Review by ghost — CHANGES_REQUESTED** [review]
Can you also address the conflicts with the latest head of branch "develop" ?

File: templates/index.hbs (hunk 3 above)
Comment (iamreyne) [reply templates/index.hbs]: I personally prefer it, but that is an easy change

File: static/css/dark.css (hunk 4 above)
Comment (iamreyne) [reply static/css/dark.css]: Really? 

I changed the text to be white, so white links + white text seems a bit weird.

If you are sure, I will change it.

File: modules/defaults.php (hunk 2 above)
Comment (iamreyne) [reply modules/defaults.php]: I feel that the Light theme aligns much more with Nano's color scheme, specifically nano.org - hence why I thought it should be default. That is an easy change, however.

File: static/css/dark.css (hunk 4 above)
Comment (ghost) [reply static/css/dark.css]: I should have been a bit more specific: When you look at the current monitor front page (http://138.197.179.164/) text color is light gray with white links, except for the "tables" with the node infos, where all is white - text and links. 

In regular text, there should be a color difference between text and links. You are right on that! I feel that the orange links are a bit too much, so I'd prefer gray vs. white. In the are where the actual node infos are being displayed, I would not want to have a color difference between text and links. 

Having the text in light gray in the current version makes the main info, the node stats, stick out a bit more in my opinion. But I am also fine with white text and maybe light gray links, i.e. switch it around. 

Let's wait for @BitDesert 's view on that ...

Comment (iamreyne) [reply static/css/dark.css]: I've changed them to grey - you can preview here: https://node.usenano.io/

I like this better

**Comment by ghost** [comment]
Shall we merge this into develop? There some minor things I want to change, but I'd like to do that after the merge.

**Review by ghost — APPROVED** [review]

**Review by BitDesert — APPROVED** [review]

**Pull request merged**